src/confdb/gui/ConfigurationTreeActions.java

```diff
@@ -3068,7 +3067,17 @@ public static boolean insertReference(JTree tree, String type, String name) {
 				return false;
 			reference = config.insertOutputModuleReference(parent, index, referencedOutput);
 		} else if (type.equalsIgnoreCase("Module")) {
-			String[] s = name.split(":");
+			// The "unlikely string" hack.
+			//  Here, the variable "name" can presumably take values of the form
+			//  "pluginType", "pluginType:moduleLabel", or "copy:pluginType:moduleLabel".
+			//  Unfortunately, this convention does not take into account the fact that
+			//  pluginType itself can contain the substring "::" (e.g. plugin types with namespace specification,
+			//  or Alpaka plugins with explicit backend selection).
+			//  The hack below consists in replacing "::" in "name" with
+			//  a very unlikely string without semicolons, before "name" is split by ":".
```

Comment:
```suggestion
			//  a very unlikely string without colons, before "name" is split by ":".
```
Typo, to be fixed before possibly merging this PR.

Reply:
Fixed in 1203e1d6c8f3b106172d2833c49258501a63e560.